src/zjs_callbacks.c

```diff
@@ -326,6 +326,15 @@ void zjs_remove_callback(zjs_callback_id id)
     }
 }
 
+void zjs_remove_all_callbacks()
+{
+   for (int i = 0; i < cb_size; i++) {
+        if (cb_map[i]) {
+            zjs_remove_callback(i);
```

Comment:
Probably not, but I don't think ring buffer was the issue I was seeing.  cb_map was the item making zjs_malloc calls.  Not saying ring buffer couldn't also be an issue however.